app/api/migrations/migrations/25-sanitize-string-geolocations/specs/fixtures.js

```diff
@@ -0,0 +1,101 @@
+/** @format */
```

Comment:
@fnocetti format not needed anymore.

Reply:
Second time it happens to me when cloning a file for reference [facepalm], I'm sorry. I'll take care of it.
Btw, might it be a good idea to remove those comments from every other file?